packages/itinerary-body/src/AccessLegBody/rented-vehicle-subheader.js

```diff
@@ -0,0 +1,76 @@
+import { configType, legType } from "@opentripplanner/core-utils/lib/types";
+import {
+  getCompaniesLabelFromNetworks,
+  getModeForPlace
+} from "@opentripplanner/core-utils/lib/itinerary";
+import React from "react";
+
+import * as Styled from "../styled";
+
+/**
+ * A component to display vehicle rental data. The word "Vehicle" has been used
+ * because a future refactor is intended to combine car rental, bike rental
```

Comment:
"because we plan to..."

Reply:
Let's just wait until this refactor actually happens to change this comment.